src/packages/@ncigdc/components/Links/InternalLink.js

```diff
@@ -27,7 +38,10 @@ const InternalLink = ({
 
   const q = removeEmptyKeys ? removeEmptyKeys(q1) : q1;
 
-  const validAttrProps = validAttributes(rest);
+  const validAttrProps = Object.assign(
+    _.pick(rest, reactRouterLinkProps),
```

Comment:
rather than combine `validAttrProps with "valid router" props, can we spread the valid router props separately?